broker/conf/broker.conf

```diff
@@ -217,6 +217,12 @@ DEFAULT_REGION_NAME=""
 # Allow region selection when creating applications. Default is true
 ALLOW_REGION_SELECTION="true"
 
+# Node Disk Buffer Percentage
+#
+# When moving a gear to a new node, ensure that after a move this percentage of free disk space will
```

Comment:
I know I suggested using the word "percentage", but now I realise this is still confusing (does a setting of "0.05" mean the buffer will be 0.05%?), so let's try to make this clearer: "percentage of free disk space, expressed as a decimal (for example, 0.05 for 5%), will be left on the new node."